R/clean_acs.R

```diff
@@ -1,4 +1,4 @@
-clean_acs <- function(geography, variables, year, span, key, ...) {
+clean_acs <- function(geography, variables, year, span, key, short = FALSE, ...) {
```

Comment:
option to shorten column names for easy renaming & coding use (keeps full meaningful name in column label)